backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -556,10 +551,27 @@ public FutureExperimentRunDAO(
                 ModelDBActionEnum.ModelDBServiceActions.READ,
                 ModelDBResourceEnum.ModelDBServiceResourceTypes.PROJECT)
             .thenApply(
-                projIds ->
-                    new QueryFilterContext()
-                        .addCondition("experiment_run.project_id in (<authz_project_ids>)")
-                        .addBind(q -> q.bindList("authz_project_ids", projIds)),
+                resources -> {
+                  List<Resources> accessAllResources =
+                      resources.stream()
+                          .filter(Resources::getAllResourceIds)
```

Comment:
Filter with this method seems very weird. Could you expand with a method that returns a boolean?

Reply:
Ok, Fixed this with helper function which return `boolean`  and check that `boolean` in if